data/tabular/MUV_466/transform.py

```diff
@@ -0,0 +1,12 @@
+import pandas as pd
+
+
+def transform_data():
+    df = pd.read_csv(
+        "https://huggingface.co/datasets/AdrianM0/MUV/raw/main/MUV_466/data_clean.csv"
```

Comment:
We can move the HF datasets then to our org. but this is not urgent: https://huggingface.co/chemNLP
Feel free to send a request to get you added to it.

Reply:
sent a request

Reply:
I still need to get admin access to add you.